rpc/client.go

```diff
@@ -20,12 +21,17 @@ import (
 	"github.com/statechannels/go-nitro/channel/state/outcome"
 )
 
+type CompletedObjectives struct {
```

Comment:
Should this be private? Since it has no functions and all it's members are private, it's not super useful outside of the `rpc` package.